api/rpm_and_image_fetcher.py

```diff
@@ -0,0 +1,72 @@
+import os
+import requests
+import logging
+import json
+
+# Set your GitHub access token as an environment variable
+GITHUB_TOKEN = os.environ.get("GITHUB_PERSONAL_ACCESS_TOKEN")
+headers = {
+    "Authorization": f"token {GITHUB_TOKEN}"
+}
+
+REPO_OWNER = "openshift-eng"
+REPO_NAME = "ocp-build-data"
+
+def get_directory_contents(branch, directory):
+    path = f"{REPO_OWNER}/{REPO_NAME}/contents/{directory}?ref={branch}"
+    contents_api = f"https://api.github.com/repos/{path}"
```

Comment:
There are python libraries that we can use, which could be a better to gather data from GitHub API, like `ghapi` for example: https://ghapi.fast.ai/

Reply:
Good to know! I will try using it to make one more version.